app/proguard.cfg

```diff
@@ -111,3 +111,6 @@
 }
 
 -keep class android.support.v7.widget.SearchView { *; }
+
+-dontwarn org.osmdroid.**
+-dontnote org.osmdroid.**
```

Comment:
Be careful with this use of Proguard. This will make Proguard ignore all osmdroid classes. Please double check it.